api/rds.proto

```diff
@@ -285,12 +285,20 @@ message RouteAction {
 }
 
 message RedirectAction {
-  // A 302 redirect response will be sent which swaps the host portion of the
-  // URL with this value.
+  // The host portion of the URL will be swapped with this value.
   string host_redirect = 1;
-  // A 302 redirect response will be sent which swaps the path portion of the
-  // URL with this value.
+  // The path portion of the URL will be swapped with this value.
   string path_redirect = 2;
+
+  enum HTTPResponseCode {
```

Comment:
From the perspective of documentation generation, this is going to appear a bit confusing on the webpage. Suggest renaming to RedirectResponseCode or something to that effect, so as to disambiguate.